manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml

```diff
@@ -103,7 +103,7 @@ spec:
 
     For quick descriptions of all the settings you can configure in the Kiali
     Custom Resource (CR), see the file
-    [kiali_cr.yaml](https://github.com/kiali/kiali-operator/blob/v1.36/deploy/kiali/kiali_cr.yaml)
+    [kiali_cr.yaml](https://github.com/kiali/kiali-operator/blob/v1.46/deploy/kiali/kiali_cr.yaml)
```

Comment:
This section (lines 104-106) is different now. 1.46 branch won't have the kiali_cr yaml - so you need to change this to look like what master looks like now: https://github.com/kiali/kiali-operator/blob/master/manifests/kiali-ossm/manifests/kiali.clusterserviceversion.yaml#L104-L106

```
    For quick descriptions of all the settings you can configure in the Kiali
    Custom Resource (CR), see
    [the kiali.io docs](https://kiali.io/docs/configuration/kialis.kiali.io/).
```